front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1871,7 +1871,7 @@ export type TrainScheduleResultData = {
   }[];
   stops?: {
     duration?: number;
-    id?: number;
+    id?: number | string;
```

Comment:
You shouldn't hard change this file. You need to update openapi_legacy.yaml

Reply:
🤦 obviously...

Reply:
done